app/components/workflows/button-bar.js

```diff
@@ -1,4 +1,20 @@
 import Ember from 'ember';
 
 export default Ember.Component.extend({
+  store: Ember.inject.service('store'),
+  actions: {
+    deleteWorkflows(){
+      var confirmText = 'Delete checked workflows?';
+      var i;
+      if(confirm( confirmText)){
+        var checked = $(":checkbox:checked");
+        for (i=0; i<checked.length; i++){
+          this.get('store').findRecord('workflow',checked[i].id).then(function(model){
+            model.deleteRecord();
+            model.save();
```

Comment:
Mudei para destroyRecord(); que no fundo faz tbm o save, mas fica mais bonito. Eu estava usando o save de acordo com a seguinte documentação do ember: Marks the record as deleted but does not save it. You must call save afterwards if you want to persist it.